### PR TITLE
[v19.x] Backport 19.3 candidates, v19.3.0 release notes and version bump

### PR DESCRIPTION
**Pull request by UdjinM6**
## Issue being fixed or feature implemented
backporting `backport-candidate-19.x` to `v19.x`

## How Has This Been Tested?
in progress...

## Breaking Changes
Should be none but there were a couple of merge conflicts, pls test

## Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation
- [x] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_

### CODE REVIEW

**Review by PastaPastaPasta — APPROVED** [review]
utACK for merge via merge commit contingent on CI; I was slightly concerned the bls commit would cause merge conflicts but it may not.. will test

File: doc/release-notes.md [hunk 1]
```diff
@@ -46,79 +36,47 @@ reindex or re-sync the whole chain.
...
-Recent changes to `dsq` and `dstx` messages allowed mobile clients that get
-masternode lists from `mnlistdiff` message to determine the masternode related
-to these messages because the `proTxHash` was used instead of the
-`masternodeOutpoint`. Once the v19 Hard Fork activates the signature of `dsq`
-and `dstx` messages will be based on the `proTxHash` which should make it
-possible for mobile clients to verify it.
+Wallets with 100k+ txes should now be able to rescan and not hang forever
+processing notifications for every tx. Running `keypoolrefill` with large number
+of keys and will no longer lockup GUI and can be interrupted. Running
+`upgradetohd` can also be interrupted now.
```
Comment (thephez) [inline doc/release-notes.md:49]:
```suggestion
Wallets with 100k+ txes should now be able to rescan without hanging forever while
processing notifications for every tx. Running `keypoolrefill` with a large number
of keys will no longer lockup the GUI and can be interrupted. Running
`upgradetohd` can also be interrupted now.
```

File: doc/release-notes.md [hunk 2]
```diff
@@ -46,79 +36,47 @@ reindex or re-sync the whole chain.
...
-`debug.log`
-- should use less memory in the long run comparing to older versions
-- gmp library detection should work better on macos
-- fixed a couple of typos
+- Added Kittywhiskers Van Gogh (kittywhiskers) and Odysseas Gabrielides
+(ogabrielides) to contributors list in 19.2.0 release notes
+- There should be no false "unknown rules activated" warning in GUI and RPCs now
+- Empty `settings.json` file no longer results in node startup failure
+- Block processing was slightly optimized
+- BLS library was updated to version 1.3.0 to fix a cople of tiny issues
```
Comment (thephez) [inline doc/release-notes.md:67]:
```suggestion
- BLS library was updated to version 1.3.0 to fix a couple tiny issues
```

**Review by thephez — COMMENTED** [review]
Minor suggestions

File: doc/release-notes.md [hunk 3]
```diff
@@ -46,79 +36,47 @@ reindex or re-sync the whole chain.
...
 
-Recent changes to `dsq` and `dstx` messages allowed mobile clients that get
-masternode lists from `mnlistdiff` message to determine the masternode related
-to these messages because the `proTxHash` was used instead of the
-`masternodeOutpoint`. Once the v19 Hard Fork activates the signature of `dsq`
-and `dstx` messages will be based on the `proTxHash` which should make it
-possible for mobile clients to verify it.
+Wallets with 100k+ txes should now be able to rescan without hanging forever
+while processing notifications for every tx. Running `keypoolrefill` with a
+large number of keys and will no longer lockup the GUI and can be interrupted.
```
Comment (thephez) [inline doc/release-notes.md:48]: Missed one :slightly_smiling_face: 
```suggestion
large number of keys will no longer lockup the GUI and can be interrupted.
```

**Review by thephez — APPROVED** [review]
Release notes ACK
It does seem that we could probably drop the DIP3 / 0.13 mentions in the "How to upgrade" section in the future?

**Review by PastaPastaPasta — APPROVED** [review]
utACK for merging via merge commit

**Comment by UdjinM6** [comment]
Reindexed with `--assumevalid=0`, no issues.

Gitian hashes
```
175c82c11a8db623fbdba5056966ef44e412a3dee08982dc250e12ba80ce8f40  dashcore-1c81bddf1c02-aarch64-linux-gnu-debug.tar.gz
f4eeca697f8aa76297a8e5396c435bb714da178322f80ec82cca7454eff52cb7  dashcore-1c81bddf1c02-aarch64-linux-gnu.tar.gz
625027cf0599a5893608d1ffa3c0a656919c405f41f9c12cc822bf643fe65554  dashcore-1c81bddf1c02-osx64-debug.tar.gz
5dc48867554b7054c86aa1eea06ef2929822b742f2092806a281c8289bdfa907  dashcore-1c81bddf1c02-osx64.tar.gz
84d94208e833f3ee705063ed31993805181b5077cfecff00381cd5160232ef24  dashcore-1c81bddf1c02-osx-unsigned.dmg
611c67a8bdcc784884ed88d4fae1e48158c7628c42b3393e89785f208e15e5be  dashcore-1c81bddf1c02-riscv64-linux-gnu-debug.tar.gz
4e87cca67b16e51fa243a83b11353f27e15d14e89a1c4962e3db67cb811b0708  dashcore-1c81bddf1c02-riscv64-linux-gnu.tar.gz
461b70b91879e95baf6d4e89652dff33bd7cd1530c4c291b970dadc289017184  dashcore-1c81bddf1c02.tar.gz
0f6745d3c4f1cf3f8e4e9dacb3507e060b95b68b1bd05845e7736ab6907cc35f  dashcore-1c81bddf1c02-win64-debug.zip
b85c176bb50f816b750d9e2c0efb7fd89196702079c06220cf61188cc12e514f  dashcore-1c81bddf1c02-win64-setup-unsigned.exe
e5a08b39ac802e300b162e1a3a11dc7c31e4ce797cea1071e43c815aa5fb88d0  dashcore-1c81bddf1c02-win64.zip
c2a3154996c4d059b0443e534025f08d6d3679cb02450f929a097766bc022998  dashcore-1c81bddf1c02-x86_64-linux-gnu-debug.tar.gz
b2d8a3d25bb959478351fb504a867d86680830a069501e91d788f479e68c8d30  dashcore-1c81bddf1c02-x86_64-linux-gnu.tar.gz
```

**Review by ogabrielides — APPROVED** [review]
LGTM

**Pull request merged**